### ISSUE TITLE
change env vars?

### ISSUE THREAD

**Issue by brianmorton** [haugene/docker-transmission-openvpn#714]
If I've created a container with a specific set of vars passed in when the container was created, is it possible to `start` the container again and update these values?

**Comment by h45rd** [comment]
Have a look at these two old stackoverflow threads. They explain and link all of the relevant issues better than I could :)
https://stackoverflow.com/questions/27812548/how-to-set-an-environment-variable-in-a-running-docker-container
https://stackoverflow.com/questions/38587741/updating-environment-variables-of-a-container-using-docker
As far as I know nothing regarding this has changed. But please correct me if I am wrong.

To summarize:
While there are some hacky solutions that could accomplish your goal, the true _docker way_ would be to just destroy and re-create the container with your updated values.
The container fs itself should never contain files that you want to store permanently. That's what volumes are for.

**Comment by brianmorton** [comment]
Thank you :)
